charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -115,9 +116,90 @@ exporters:
 {{- if .Values.standaloneCollector.enabled }}
 service:
   pipelines:
+    logs:
+      exporters: [otlp]
     metrics:
       exporters: [otlp]
     traces:
       exporters: [otlp]
 {{- end }}
 {{- end }}
+
+{{- define "opentelemetry-collector.agent.containerLogsConfig" -}}
+{{- if .Values.agentCollector.containerLogs.enabled }}
+receivers:
+  filelog:
+    include: [ /var/log/pods/*/*/*.log ]
+    # Exclude collector container's logs. The file format is /var/log/pods/<namespace_name>_<pod_name>_<pod_uid>/<container_name>/<run_id>.log
```

Comment:
I think it should be configurable from values.yaml. 
k8s admin would may like to ingest otel agents' logs to monitor them after they disable `logging` exporter.

Reply:
Let's make it a separate PR if that's OK?

We probably need to prevent a scenario when user changes the property and inadvertently causes the looping of logs from exporter back into the receiver. It needs to be assured that changing of such property alone does not cause the loop.

Reply:
Sounds good, let's do in a separate PR.